README.asc

```diff
@@ -11,16 +11,18 @@ For more information, see http://cassandra.apache.org/[the Apache Cassandra web
 
 Issues should be reported on https://issues.apache.org/jira/projects/CASSANDRA/issues/[The Cassandra Jira].
```

Comment:
@pauloricardomg this whole file should be called "README.adoc". "asc" suffix does not make the document rendered in IDEA automatically.